src/pages/ReportSettingsPage.js

```diff
@@ -0,0 +1,236 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {View, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+import styles from '../styles/styles';
+import compose from '../libs/compose';
+import Navigation from '../libs/Navigation/Navigation';
+import * as Report from '../libs/actions/Report';
+import * as ReportUtils from '../libs/reportUtils';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import ScreenWrapper from '../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import Text from '../components/Text';
+import Button from '../components/Button';
+import TextInputWithPrefix from '../components/TextInputWithPrefix';
+import Picker from '../components/Picker';
+import withFullPolicy, {fullPolicyDefaultProps, fullPolicyPropTypes} from './workspace/withFullPolicy';
+
+
+const propTypes = {
+    /* Onyx Props */
```

Comment:
These should be at the bottom, right? What about `personalDetails`?

Reply:
Ah I'm realizing we don't use personalDetails here anymore. I moved these around and removed that.